packages/jest-runtime/src/index.js

```diff
@@ -29,13 +29,14 @@ import {run as cilRun} from './cli';
 import {options as cliOptions} from './cli/args';
 
 type Module = {|
```

Comment:
This should be `type Module = typeof module` to get the type definitions from flow itself (it would have yelled at us for missing `loaded`, for instance). A bit more work though as it also doesn't like that `require` is added the way it is in this PR. Can revisit later